src/client.cpp

```diff
@@ -35,6 +35,11 @@
 
 TicksTime s_startTime, s_endTime;
 
+#ifdef NO_TIMERS
```

Comment:
Could we use something more descriptive than `NO_TIMERS`? Sleep is used here to act as a timer so the name is a bit confusing.

Reply:
True, I changed it to NO_SETITIMERS_SYSCALL since that's exactly what we are doing here, if a better name does come to mind please do say so and I'll change it

Reply:
Names sounds good to me 👍